src/components/IntegrationsView/JSONFormStyles.css

```diff
@@ -0,0 +1,29 @@
+fieldset {
```

Comment:
is this file used anywhere?

Reply:
Those ones should be removed, it's a legacy from the json schema parsing library